### DISCUSSION TITLE
Omega Alpha 3

### DISCUSSION THREAD

**Discussion by xyzfre** [jurialmunkey/skin.arctic.horizon.2#982] [Discussion]
Has anyone tried Alpha 3 on Windows 11 ,AH2 ,I tried it yesterday and noticed strange behavior on widgets progress icons  and episode count

**Comment by xyzfre** [comment]
@jurialmunkey is Omega 3 compatible with AH2

**Reply by jurialmunkey** [reply]
No idea. Haven't used AH2 in a while as Fuse is my daily driver.  Been using Alpha 3 on my Windows laptop since it was released and haven't had any major issues with Fuse there. I can't see any reason for AH2 to have any issues but can't say for certain since I haven't used it much at all lately.

**Reply by xyzfre** [reply]
I LOVE AH2 you've done some AMAZING work ,I reverted back to K20.2 I hope you get a chance to try it out please ,,Thank you For creating a skin, so perfect and unique

**Reply by xyzfre** [reply]
@jurialmunkey I tried 21 Alpha 3 but it’s giving me issues with AH2 the widgets progress icons and episode count and next page icon look distorted

**Reply by jurialmunkey** [reply]
Looks fine to me

![screenshot00137](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/4618719/23fbc61c-11cb-4274-a104-d8a18c95433b)

**Reply by xyzfre** [reply]
Weird maybe I should try to do a fresh install of Alpha 3 instead of updating over 20.2 thank you @jurialmunkey for checking really appreciated

**Reply by xyzfre** [reply]
@jurialmunkey I figured out where my glitch is coming from in Alpha 3 if you set a limit on the home screen widgets For example, 20 The next page icon gets distorted if you set it to None, everything displays proper

**Reply by xyzfre** [reply]
I forgot to mention in settings ,Menus , Limited widgets, have more items object also has to be enabled and then you will see the glitch take place

**Reply by xyzfre** [reply]
https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/49699869/8a7efe28-5e5f-40ed-b396-c538aace21b4

**Reply by xyzfre** [reply]
My apologies @jurialmunkey it seems to be happening with Fen widgets I have 2 different set ups on 2 PC's one running TMDB widgets the other Fen widgets

**Reply by xyzfre** [reply]
> https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/49699869/8a7efe28-5e5f-40ed-b396-c538aace21b4

When you scroll to the end of the roll of widgets it glitches out

**Reply by xyzfre** [reply]
![image](https://github.com/jurialmunkey/skin.arctic.horizon.2/assets/49699869/b95d00fd-5a92-45d2-99cf-8c44497fb792)

**Reply by xyzfre** [reply]
@jurialmunkey this is how the progress icons look when using Fen ,, if there's anything you could do, I would really really appreciate your help this only happens on Alpha 3

**Reply by jurialmunkey** [reply]
> I forgot to mention in settings ,Menus , Limited widgets, have more items object also has to be enabled and then you will see the glitch take place

Disable the more items object in the skin. Kodi now provides this behaviour natively in Omega.

> this is how the progress icons look when using Fen ,, if there's anything you could do, I would really really appreciate your help this only happens on Alpha 3

There have been no relevant skin engine changes which would cause this behaviour so there is nothing I can fix here. I also cannot reproduce it using the library or add-ons which are not banned.

It is either a problem with the addon, Kodi itself, or something in your configuration.

**Reply by xyzfre** [reply]
> > I forgot to mention in settings ,Menus , Limited widgets, have more items object also has to be enabled and then you will see the glitch take place
> 
> 
> 
> Disable the more items object in the skin. Kodi now provides this behaviour natively in Omega.
 @jurialmunkey thank you for this info
> 
> 
> > this is how the progress icons look when using Fen ,, if there's anything you could do, I would really really appreciate your help this only happens on Alpha 3
> 
> 
> 
> There have been no relevant skin engine changes which would cause this behaviour so there is nothing I can fix here. I also cannot reproduce it using the library or add-ons which are not banned.
> 
> 
> 
> It is either a problem with the addon, Kodi itself, or something in your configuration.
> 
> 
> 
> 

I don't think it's something in my configurations. I tried on two different PCs. and one of them was a fresh install same behavior ,,for TMDB The progress icons display correctly,, I think something has changed in Alpha 3

**Reply by jurialmunkey** [reply]
The skin hasn't changed so it cannot be the skin. There also have not been any significant skinning engine changes which would affect this type of thing.

That means that either the addon has changed or Kodi has changed or your configuration has changed. None of these things are things which I control.

**Reply by xyzfre** [reply]
Thank you @jurialmunkey this is been killing me. I tried a fresh install two different devices same behavior. I check with Arctic Fuse but it also displays the progress,episode count icons with the same behavior

**Reply by xyzfre** [reply]
https://github.com/xbmc/xbmc/pull/23841

**Reply by xyzfre** [reply]
@jurialmunkey I think this commit here is what's causing the issue or the changes
